pkgs/games/steam/runtime.nix

```diff
@@ -1,17 +1,25 @@
-{ stdenv, steamArch, fetchurl, writeText, python2, dpkg }:
+{ stdenv, steamArch, fetchurl, }:
 
-let input = builtins.getAttr steamArch (import ./runtime-generated.nix { inherit fetchurl; });
+stdenv.mkDerivation rec {
 
-    inputFile = writeText "steam-runtime.json" (builtins.toJSON input);
+  name = "steam-runtime";
+  # from https://repo.steampowered.com/steamrt-images-scout/snapshots/
+  version = "0.20200417.0";
 
-in stdenv.mkDerivation {
-  name = "steam-runtime-2016-08-13";
-
-  nativeBuildInputs = [ python2 dpkg stdenv.cc.bintools ];
+  src =
+    if steamArch == "amd64" then fetchurl {
```

Comment:
I think steamArch might be subtly different now. amd64 now means download the amd64+i386 runtime snapshot because there isn't an x64-only one.

Before, iiuc, the `steam-chrootenv` would go out of its way to get the i386 runtime for amd64 since amd64 didn't already include it. It seems like we can probably simplify that code in `default.nix` for the chrootenv a tiny bit now too!